plugin/pkg/admission/resourcequota/apis/resourcequota/install/install.go

```diff
@@ -22,12 +22,12 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 	utilruntime "k8s.io/apimachinery/pkg/util/runtime"
 	resourcequotaapi "k8s.io/kubernetes/plugin/pkg/admission/resourcequota/apis/resourcequota"
-	resourcequotav1alpha1 "k8s.io/kubernetes/plugin/pkg/admission/resourcequota/apis/resourcequota/v1alpha1"
+	resourcequotav1beta1 "k8s.io/kubernetes/plugin/pkg/admission/resourcequota/apis/resourcequota/v1beta1"
 )
 
 // Install registers the API group and adds types to a scheme
 func Install(scheme *runtime.Scheme) {
 	utilruntime.Must(resourcequotaapi.AddToScheme(scheme))
-	utilruntime.Must(resourcequotav1alpha1.AddToScheme(scheme))
```

Comment:
I think you should have this version but setting the default version to beta in subsequent line which you are doing should be good enough.

Reply:
actually both should be added in subsequent line in priority order. will update, Thanks!